src/qt/multisenddialog.cpp

```diff
@@ -70,9 +70,11 @@ void MultiSendDialog::on_viewButton_clicked()
     std::pair<std::string, int> pMultiSend;
     std::string strMultiSendPrint = "";
     if (pwalletMain->isMultiSendEnabled()) {
-        if (pwalletMain->fMultiSendStake)
-            strMultiSendPrint += "MultiSend Active for Stakes\n";
+        if (pwalletMain->fMultiSendStake && pwalletMain->fMultiSendMasternodeReward)
+            strMultiSendPrint += "MultiSend Active for Stakes and Masternode Rewards\n";
         else if (pwalletMain->fMultiSendStake)
+            strMultiSendPrint += "MultiSend Active for Stakes\n";
+        else if (pwalletMain->fMultiSendMasternodeReward)
             strMultiSendPrint += "MultiSend Active for Masternode Rewards\n";
```

Comment:
nit: I think I would go for 
if (a) print a
if (b) print b
instead of 
if (a&b) print a & b
elif ...